addons/binding/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/model/ChannelGroup.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.nibeuplink.internal.model;
+
+/**
+ * Sensor = readonly, Setting = read/write
+ *
+ * @author Alexander Friese - initial contribution
+ *
+ */
+public enum ChannelGroup {
+    Sensor,
```

Comment:
Same here: Java enum's should be in uppercase.

Reply:
fixed it